recipes/repo2rocrate/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "repo2rocrate" %}
+{% set version = "0.1.2" %}
+{% set python_min = "3.9" %}  # Define the minimum Python version
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/repo2rocrate-{{ version }}.tar.gz
+  sha256: ae120c35180f852a18dee8c6ea114dc0c65abaa111b26a3da87d4f211dd7eedc
+
+build:
+  entry_points:
+    - repo2rocrate=repo2rocrate.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - pip
+  run:
+    - python >={{ python_min }}
+    - pyyaml
+    - click
+    - rocrate
+
+test:
+  imports:
+    - repo2rocrate
+  commands:
+    - repo2rocrate --help
+  requires:
+    - python {{ python_min }}
+
+about:
+  home: https://github.com/crs4/repo2rocrate
+  summary: Generate a Workflow RO-Crate from a best-practice workflow repository 
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - simleo
```

Suggest removing @simleo due to comment on Slack about not being familiar with conda-forge (though happy to keep here - more the merrier! Just need a comment on the PR to approve).

Adding @mirpedrol and myself here to give redundancy.

```suggestion
    - mirpedrol
    - ewels
```